cookbooks/php/recipes/default.rb

```diff
@@ -0,0 +1,6 @@
+include_recipe "php::install"
+include_recipe "php::configure"
+include_recipe "php::composer"
+if ['app_master', 'app', 'solo'].include? node['dna']['instance_role']
```

Comment:
`php` recipe runs only on app instances, therefore this if is not required

Reply:
The php recipe runs on utility instances too. `php` is added to the list of recipes on dna.json which is then called on `app::create`.